.emacs.d/lisp/code/languages/rust.el

```diff
@@ -16,4 +16,10 @@
 ;; Don't indent `where' clauses in Rust.
 (setf rust-indent-where-clause nil)
 
+;; Enable LSP (to talk to RLS) on all rust-mode buffers
+;;
+;; NOTE This could get quite annoying if you're not aiming to use RLS
+(require 'lsp-mode)
+(add-hook #'rust-mode-hook #'lsp)
+
```

Comment:
Remove these, as they are opinionated. (Not everyone would want LSP in Rust, necessarily; this should be turned on by users.)